website/docs/sql_queries.md

```diff
@@ -362,37 +349,37 @@ Following tables show whether a given query is supported on specific query engin
 
 ### Copy-On-Write tables
 
-| Query Engine          |Snapshot Queries|Incremental Queries|
-|-----------------------|--------|-----------|
-| **Hive**              |Y|Y|
-| **Spark SQL**         |Y|Y|
-| **Flink SQL**         |Y|N|
-| **PrestoDB**          |Y|N|
-| **Trino**             |Y|N|
-| **AWS Athena**        |Y|N|
-| **BigQuery**          |Y|N|
-| **Impala**            |Y|N|
-| **Redshift Spectrum** |Y|N|
-| **Doris**             |Y|N|
-| **StarRocks**         |Y|N|
-| **ClickHouse**        |Y|N|
+| Query Engine          | Snapshot Queries | Incremental Queries |
+|-----------------------|------------------|---------------------|
+| **Hive**              | Y                | Y                   |
+| **Spark SQL**         | Y                | Y                   |
+| **Flink SQL**         | Y                | N                   |
+| **PrestoDB**          | Y                | N                   |
+| **Trino**             | Y                | N                   |
+| **AWS Athena**        | Y                | N                   |
+| **BigQuery**          | Y                | N                   |
+| **Impala**            | Y                | N                   |
+| **Redshift Spectrum** | Y                | N                   |
+| **Doris**             | Y                | N                   |
+| **StarRocks**         | Y                | Y                   |
```

Comment:
Are incremental queries supported in starrocks?

Reply:
No, I ran the example mentioned [here](https://github.com/StarRocks/demo/tree/master/documentation-samples/hudi) incremental queries are not supported in both MOR and COW.